u_tls_extensions.go

```diff
@@ -675,6 +675,8 @@ FAKE EXTENSIONS
 */
 
 type FakeChannelIDExtension struct {
+	// The extension ID changed from 30031 to 30032. Set to true to use the old extension ID.
+	OldExtensionID bool
```

Comment:
Is this backwards compatible API-wise? I personally wouldn't care, but I think Sergey will

Reply:
Yes.  Any current instantiations of `FakeChannelIDExtension` would have to be of the form `FakeChannelIDExtension{}` or `new(FakeChannelIDExtension)`.  Both of these will now result in a value with `OldExtensionID` set to false, which results in the existing behavior (the new extension ID is written to the input in `FakeChannelIDExtension.Read`)

Reply:
aha ok great thanks for being patient with me there and walking me through that!